app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -145,15 +148,25 @@ class BloodPressureHistoryScreen : BaseScreen<
       container: ViewGroup?
   ) = ScreenBpHistoryBinding.inflate(layoutInflater, container, false)
 
+  override fun onAttach(context: Context) {
+    super.onAttach(context)
+    context.injector<BloodPressureHistoryScreenInjector>().inject(this)
```

Comment:
Rename the `BloodPressureHistoryScreenInjector` to `Injector` to maintain consistency